tools/annovar-explicit.cwl

```diff
@@ -133,6 +134,7 @@ arguments:
 
 inputs:
   cache: { type: File, doc: "TAR GZ file with RefGene, KnownGene, and EnsGene reference annotations" }
+  ram: {type: int?, default: 32000, doc: "May need to increase this value depending on the size/complexity of input"}
```

Comment:
Can never remember if unquoted `int?` passes cwltool validation.

Reply:
Test and passed